dcql/src/dcql-query/dcql-query-complex.test.ts

```diff
@@ -208,510 +208,410 @@ describe('complex-mdoc-query', () => {
     ])
 
     assert.deepStrictEqual(res, {
-      credentials: complexMdocQuery.credentials,
```

Comment:
This test is a bit instance and I'm not sure how much it adds as it's basically impossible to update. After spending over 20 minutes i just copied the new output, but since it's over 2000 lines there's no way to correctly check it. We should probably reduce the test to what is really import to cover